Observer/TescoPaymentObserver.php

```diff
@@ -0,0 +1,99 @@
+<?php
+namespace Omise\Payment\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Payment\Model\InfoInterface;
```

Comment:
@jacstn Guess we don't need this interface (don't see it's being used in this class)?